### PR TITLE
Feature/compare siard files in i ts

### PR DESCRIPTION
**Pull request by sdbuehlmann**
Until now, the generated archives have not been audited in ITs. With this change, at least some information from the header/metadata.xml files are compared between an expected and an actual SIARD archive.

### CODE REVIEW

File: src/integrationTest/java/ch/admin/bar/siard2/cmd/utils/siard/UnzipperTest.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import java.io.IOException;
+
+public class UnzipperTest {
+
+    @Rule
+    public TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+    @Test
+    public void unzip_expectUnzippedArchive() throws IOException {
```
Comment (StephGit) [inline src/integrationTest/java/ch/admin/bar/siard2/cmd/utils/siard/UnzipperTest.java:19]: ``
```suggestion
    public void shouldUnzipArchive() throws IOException {
```

File: src/integrationTest/java/ch/admin/bar/siard2/cmd/utils/siard/model/StringWrapper.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    @Override
+    public String toString() {
+        return value;
+    }
+
+    @Override
+    public StringWrapper applyUpdates(Updater updater) {
+        val updatedThis = updater.applyUpdate(this);
```
Comment (StephGit) [inline src/integrationTest/java/ch/admin/bar/siard2/cmd/utils/siard/model/StringWrapper.java:27]:
```suggestion
       return updater.applyUpdate(this);
```

File: src/integrationTest/java/ch/admin/bar/siard2/cmd/utils/siard/model/StringWrapper.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+        return updatedThis;
```
Comment (StephGit) [inline src/integrationTest/java/ch/admin/bar/siard2/cmd/utils/siard/model/StringWrapper.java:29]: ```suggestion```

File: src/integrationTest/java/ch/admin/bar/siard2/cmd/utils/siard/model/StringWrapper.java (hunk 2 above)
Comment (StephGit) [reply src/integrationTest/java/ch/admin/bar/siard2/cmd/utils/siard/model/StringWrapper.java:27]: Inline variable

File: src/integrationTest/java/ch/admin/bar/siard2/cmd/utils/siard/UnzipperTest.java (hunk 1 above)
Comment (sdbuehlmann) [reply src/integrationTest/java/ch/admin/bar/siard2/cmd/utils/siard/UnzipperTest.java:19]: Thanks for your comment, @StephGit. I prefer using the 'calledMethod_contextInformation_expectedResult' pattern for unit tests. If there isn't a specific convention that requires a different pattern, I would like to stick with my current solution. I hope that's okay with you.

**Pull request merged**